_data-prepper/pipelines/configuration/processors/parse-xml.md

```diff
@@ -0,0 +1,55 @@
+---
+layout: default
+title: parse_xml
+parent: Processors
+grand_parent: Pipelines
+nav_order: 83
+---
+
+# parse_xml
+
+The `parse_xml` processor parses XML data for an event.
+
+## Configuration
+
+You can configure the `parse_xml` processor with the following options.
+
+| Option | Required | Type | Description |
+| :--- | :--- | :--- | :--- | 
+| `source` | No | String | The field in the `event` that is parsed. Default value is `message`. |
+| `destination` | No | String | The destination field of the parsed XML. Defaults to the root of the `event`. Cannot be `""`, `/`, or any white-space-only `string` because these are not valid `event` fields. |
+| `pointer` | No | String | A JSON pointer to the field to be parsed. There is no `pointer` by default, meaning that the entire `source` is parsed. The `pointer` can access JSON array indexes as well. If the JSON pointer is invalid, then the entire `source` data is parsed into the outgoing `event`. If the key that is pointed to already exists in the `event` and the `destination` is the root, then the pointer uses the entire path of the key. |
```

Comment:
Does "no pointer by default" mean "the default pointer is empty? Null?

Reply:
Null, fixed.